lib/cc-reno.c

```diff
@@ -28,49 +28,61 @@
 void quicly_cc_init(quicly_cc_t *cc, uint32_t initcwnd)
 {
     memset(cc, 0, sizeof(quicly_cc_t));
-    cc->cwnd = initcwnd;
-    cc->ssthresh = UINT32_MAX;
+    cc->type = CC_RENO_MODIFIED;
+    cc->cwnd = cc->cwnd_initial = initcwnd;
```

Comment:
Is it intentional to not set `cwnd_maximum` here? If that is the case, it might be a good idea to clarify that in the doc-comment of `cwnd_maximum`.

Reply:
Oh, good catch. Not intentional, fixing now.